content/200-concepts/050-overview/300-prisma-in-your-stack/01-rest.md

```diff
@@ -141,11 +141,25 @@ app.delete(`/post/:id`, async (req, res) => {
 
-| Example                                                                                                             | Language   | Stack        | Description                                                         |
-|---------------------------------------------------------------------------------------------------------------------|------------|--------------|---------------------------------------------------------------------|
-| [`rest-nextjs-api-routes`](https://github.com/prisma/prisma-examples/tree/latest/typescript/rest-nextjs-api-routes) | TypeScript | Fullstack    | Simple [Next.js](https://nextjs.org/) app (React) with a REST API   |
-| [`rest-express`](https://github.com/prisma/prisma-examples/tree/latest/typescript/rest-express)                     | TypeScript | Backend only | Simple REST API with Express                                        |
-| [`rest-nextjs`](https://github.com/prisma/prisma-examples/tree/latest/javascript/rest-nextjs)                       | JavaScript | Fullstack    | Simple [Next.js](https://nextjs.org/) app (React) with a REST API   |
-| [`rest-nestjs`](https://github.com/prisma/prisma-examples/tree/latest/typescript/rest-nestjs)                       | TypeScript | Backend only | Simple [Nest.js](https://nestjs.com/) app (Express) with a REST API |
+### TypeScript
+
+| **Example**            | **Stack**    | **Description**                                 |
+|------------------------|--------------|-------------------------------------------------|
+| rest-express           | Backend only | REST API with Express for TypeScript            |
+| rest-fastify           | Backend only | REST API using Fastify and Prisma Client.       |
+| rest-hapi              | Backend only | REST API using hapi and Prisma Client           |
+| rest-nestjs            | Backend only | Nest.js app (Express) with a REST API           |
+| rest-nextjs-express    | Fullstack    | Next.js app (React, Express) and Prisma Client  |
+| rest-nextjs-api-routes | Fullstack    | Next.js app (React) with a REST API             |
+| rest-nextjs-api-routes | Fullstack    | Implement authentication using NextAuth.js      |
+
+### JavaScript
+
+| **Example**  | **Stack**    | **Description**                                                  |
+|--------------|--------------|------------------------------------------------------------------|
+| rest-express | Backend only | REST API using Express and Prisma Client                         |
+| rest-fastify | Backend only | REST API using Fastify and Prisma Client                         |
+| rest-nextjs  | Fullstack    | Next.js app (React) with a REST API                              |
+| rest-nuxjs   | Fullstack    | App with NuxtJs using Vue (frontend), Express, and Prisma Client |
```

Comment:
Did you intentionally remove the links from the examples here? Personally, I like the ability to be able to navigate to an example directly from this table if I find it looks interesting.

(Also, definitely a personal preference again, but I also like rendering the example names as inline code using backticks. I always tend to do this [for file/folder names](https://www.prisma.io/docs/about/prisma-docs/style-guide#use-inline-code-format-for-paths-and-file-names), and in this case the examples are named after the folders they can be found in).

Reply:
True, I didn't yet take the time to add links to each one, but plan to do so in a future PR... it will be quite labour-intensive so I wanted to get the accurate table out first. And Yes, will for sure add the backticks.

Reply:
I just want to throw my 2 cents here. When I work with tables and markdown, I usually use excel or sheets to build my tables. Then I export the thing to CSV, open the file in VSCode, then make it a markdown table with this [extension](https://marketplace.visualstudio.com/items?itemName=Marchiore.csvtomarkdown). Hope it helps :)

Reply:
Thank you @millsp !!  I was using a tool called [Table Generator](https://www.tablesgenerator.com/markdown_tables#), but it was a bit dangerous because if you made a single mistake there was no "back" option. Niko also showed Lucy and me a cool process in VS Code for quickly selecting similar text, then adding links a batch at a time. Productivity for the win.